SpywareFilter/sections/tracking_servers.txt

```diff
@@ -3001,6 +3005,7 @@
 ||oranges88.com^$third-party
 ||orcapia.com^$third-party
 ||os-data.com^$third-party
+||ourstats.de^$third-party
```

Comment:
We have this rule.

Reply:
I don't remember adding it, seems like an accident.